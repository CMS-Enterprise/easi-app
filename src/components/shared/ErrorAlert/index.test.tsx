import React from 'react';
import { render } from '@testing-library/react';

import { ErrorAlert, ErrorAlertMessage } from './index';

describe('The ErrorAlert component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <ErrorAlert heading="test heading">
        <ErrorAlertMessage message="Message 1" errorKey="Error 1" />
        <ErrorAlertMessage message="Message 2" errorKey="Error 2" />
        <ErrorAlertMessage message="Message 3" errorKey="Error 3" />
        <ErrorAlertMessage message="Message 4" errorKey="Error 4" />
        <ErrorAlertMessage message="Message 5" errorKey="Error 5" />
      </ErrorAlert>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
