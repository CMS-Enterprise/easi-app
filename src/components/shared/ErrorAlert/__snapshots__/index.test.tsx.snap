// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`The ErrorAlert component > matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="usa-alert usa-alert--error"
    role="alert"
  >
    <div
      class="usa-alert__body"
    >
      <h3
        class="usa-alert__heading"
        tabindex="-1"
      >
        test heading
      </h3>
      <button
        class="usa-error-message usa-alert__text easi-error-alert__message"
        type="button"
      >
        Message 1
      </button>
      <button
        class="usa-error-message usa-alert__text easi-error-alert__message"
        type="button"
      >
        Message 2
      </button>
      <button
        class="usa-error-message usa-alert__text easi-error-alert__message"
        type="button"
      >
        Message 3
      </button>
      <button
        class="usa-error-message usa-alert__text easi-error-alert__message"
        type="button"
      >
        Message 4
      </button>
      <button
        class="usa-error-message usa-alert__text easi-error-alert__message"
        type="button"
      >
        Message 5
      </button>
    </div>
  </div>
</DocumentFragment>
`;
